storybookjs/storybook
# FIX styling of the Addon Info propTables

Issue: #5278

## What I did

- [x] Made the existing stub table components into their own component files
- [x] Added a stylesheet for the table (this may not be something you want me to, please let me know. I'm very happy to change my approach, especially considering that this is my first contribution to Storybook)
- [ ] Update existing snapshots (for some reason unrelated tests are failing blocking my ability to update the snapshots for the time being.

### Screenshot:
<img width="787" alt="screen shot 2019-02-05 at 3 41 48 am" src="https://user-images.githubusercontent.com/10335665/52240735-cceb1d00-2925-11e9-886e-9a9d2857bc3a.png">


## How to test
Just launch storybook and head to the button component which has a story with the Info addon. 

- Is this testable with Jest or Chromatic screenshots?
Yes
- Does this need a new example in the kitchen sink apps?
No
- Does this need an update to the documentation?
No

If your answer is yes to any of these, please make sure to include it in your PR.
